src/locales/source/lt.json

```diff
@@ -0,0 +1,258 @@
+{
+  "btn.atgal": "Atgal",
```

Comment:
The key have to stay original, only value can be translated.

Reply:
Reverted key translations...